docs/it/token_endpoint.rst

```diff
@@ -224,6 +224,11 @@ ID Token
 ++++++++
 
 L'ID Token è un JSON Web Token (JWT) che contiene informazioni sull'utente che ha eseguito l'autenticazione. I RP DEVONO eseguire la validazione dell'ID Token.
+
+.. admonition:: |cieid-icon|
+
+  Il RP PUÒ richiedere che L'ID Token sia cifrato (vedere il parametro *id_token_encrypted_response_alg* nel :ref:`Metadata RP <MetadataRP>` ). In questo caso l'ID Token sarà un JWT **firmato e cifrato** e DEVE contenere il parametro *cty* (Content-Type) nell'intestazione JOSE con il valore *JWT* (vedere :rfc:`7519#section-5.2`).
```

Comment:
Modificare in 
 Il RP PUÒ richiedere che L'ID Token sia cifrato (vedere il parametro *id_token_encrypted_response_alg* nel :ref:`Metadata RP <MetadataRP>` ). 
Se il RP inserisce nel suo metadata il parametro *id_token_encrypted_response_alg*, l'OP DEVE restituire l'ID Token cifrato. 
In questo caso l'ID Token sarà un JWT **firmato e cifrato** e DEVE contenere il parametro *cty* (Content-Type) nell'intestazione JOSE con il valore *JWT* (vedere :rfc:`7519#section-5.2`).

Reply:
https://github.com/italia/spid-cie-oidc-docs/pull/55/commits/0e038ebd85305c7f69305fc305dded8379de4022